ingestion/src/metadata/ingestion/source/dashboard/qlikcloud/models.py

```diff
@@ -0,0 +1,110 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""
+Metabase Models
+"""
+from typing import List, Optional, Union
+
+from pydantic import BaseModel
+
+
+# App
+class QlikApp(BaseModel):
+    """
+    Metabase dashboard model
+    """
+
+    description: Optional[str]
+    name: str
+    id: str
+    resourceId: str
+
+
+class QlikAppList(BaseModel):
+    apps: Optional[List[QlikApp]]
+
+
+class QlikAppDetails(BaseModel):
+    """
+    Metabase app details model
+    """
+
+    description: Optional[str]
+
+    name: Optional[str]
+    id: str
+    collection_id: Optional[str]
+
+
+# Sheet
+class QlikSheetInfo(BaseModel):
+    qId: str
+
+
+class QlikSheetMeta(BaseModel):
+    title: Optional[str]
+    description: Optional[str]
+
+
+class QlikSheet(BaseModel):
+    qInfo: QlikSheetInfo
+    qMeta: Optional[QlikSheetMeta] = QlikSheetMeta()
+
+
+class QlikSheetItems(BaseModel):
+    qItems: Optional[List[QlikSheet]] = []
+
+
+class QlikSheetAppObject(BaseModel):
+    qAppObjectList: Optional[QlikSheetItems] = QlikSheetItems()
```

Comment:
not sure if the linter will like this that much. We can use `alias` when defining the `Field` to differentiate how this gets parsed vs. how we can use it